CommonTools/Statistics/interface/SequentialCombinationGenerator.h

```diff
@@ -99,7 +99,7 @@ template<class T> class SequentialCombinationGenerator {
       }
       int mincnew2=cnew2[0];
       if (ss[n1-1]<mincnew2) return empty;
-      Vecint::iterator j1=find_if(ss.begin(),ss.end(),bind2nd(std::greater<int>(),mincnew2));
+      Vecint::iterator j1=find_if(ss.begin(),ss.end(), [&](auto c) { return c > mincnew2;});
```

Comment:
Cool, I see you are also working on removing things that are unsupported by C++17! I have a big PR in the pipeline where I work on the same issue: https://github.com/cms-sw/cmssw/pull/24549.

Just a remark: maybe it's better is the lambda function is not capturing all the local variables possible, but only what you actually use, i.e.
```
[&mincnew2](auto c) { return c > mincnew2;}
```
I think it's also possible to capture by value I think:
```
[=mincnew2](auto c) { return c > mincnew2;}
```

Reply:
How soon will your branch go in? I defer to your changes.

Reply:
Hi! I have absolutely no idea since it's still testing I got not much feedback on that PR so far. But I'm sure not by the end of this week! I'm anticipating a lot of suggestions.

Reply:
The compiler accepted the first statement. The second statement required this change:
[=, &mincnew2](auto c) { return c > mincnew2;}
I updated this branch with the first suggested change.